.circleci/config.yml

```diff
@@ -11,7 +11,9 @@ commands:
           name: install base dependencies
           command: |
               apt update
-              apt install -y build-essential libcurl4-gnutls-dev libgnutls28-dev \
+              # libcurl4-gnutls-dev libgnutls28-dev can be remove later, but for now they are used for the migration tests
```

Comment:
I'm a bit confused by where these are still used and why we should remove them if we actually need them.

Reply:
During migration we checkout the master branch and install it (see [here](https://github.com/emissions-api/emissions-api/blob/9537e7d56b79af9b67fe050ab434d3695591412c/.circleci/config.yml#L110)) to have a base database on which we can test our alembic migration steps.

So before this pull request is merged, the master still needs to install `pycurl` and therefore needs `libcurl4-gnutls-dev libgnutls28-dev`.

After we merged this pull request, the master does not have a dependency on pycurl and so we can remove those system packages from the tests.

The underlying problem is that we are tracking the python dependencies, but not the system dependencies in the version control system.